www/themes/borg/template.php

```diff
@@ -271,7 +271,7 @@ function borg_preprocess_node(&$variables){
       $footer_links['version']['data'] = backdrop_render($version);
 
       // Add the latest release date.
-      $date = format_date($release->node_created, 'short');
+      $date = format_date($release->node_project_release_supported_versions_created, 'short');
```

Comment:
This change is going to be (accidentally) reverted on the next update of the borg theme if they aren't also repeated in https://github.com/backdrop-contrib/borg.